pkg/drainer/drainer.go

```diff
@@ -28,50 +28,59 @@ import (
 
 var drainHelper *drain.Helper
 var node *corev1.Node
+var nthConfig config.Config
```

Comment:
As a best practice, it's best to have as few global state variables as possible. Your code is more extensible (and testable) when you pass the config object to functions like `getDrainHelper()`. You can test the configuration scenarios more easily (in parallel) and thoroughly when you can construct new config objects and pass those into functions instead of relying on global state.

Reply:
Refactored drainer file